src/Quartz.Extensions.DependencyInjection/Simpl/MicrosoftDependencyInjectionJobFactory.cs

```diff
@@ -15,13 +15,16 @@ public class MicrosoftDependencyInjectionJobFactory : PropertySettingJobFactory
     {
         private readonly IServiceProvider serviceProvider;
         private readonly IOptions<QuartzOptions> options;
+        private readonly JobActivatorCache activatorCache;
 
         public MicrosoftDependencyInjectionJobFactory(
             IServiceProvider serviceProvider,
-            IOptions<QuartzOptions> options)
+            IOptions<QuartzOptions> options,
+            JobActivatorCache activatorCache)
```

Comment:
OK this probably makes it harder to change JobActivatorCache internal

Reply:
Since the job factory is registered as singleton anyway, I can either fully inline JobActivatorCache into it or new up the cache in the job factory's ctor/field initializer and then make it internal. Thoughts?

Reply:
I think that's a good idea 👍🏻